awscli/customizations/scalarparse.py

```diff
@@ -77,6 +77,4 @@ def add_timestamp_parser(session):
 
 
 def add_scalar_parsers(session, **kwargs):
-    factory = session.get_component('response_parser_factory')
```

Comment:
Would we add the binary input handler logic here? If not, it does not make sense to keep this module as `scalarparse.py` and instead switch to something timestamp specific.

Reply:
I didn't put it here because binary format changes the input and output. I suppose it makes sense to change this to be timestamp specific now.